templates/.gitignore

```diff
@@ -0,0 +1,44 @@
+.gitignore
+
+# Packages
+dist
+bin
+var
+sdist
+target
+
+# Unit test / coverage reports
+.coverage
+.tox
+nosetests.xml
+
+# Translations
+*.mo
+
+# Mr Developer
+.mr.developer.cfg
+
+.metadata
+.project
+.pydevproject
+*.iml
+.idea
+.settings
+.DS_Store
+.classpath
+
+# Built documentation
+docs/
+
+
+# Wheel directory used in Travis builds.
+gcloud-java-wheels/
+src/test/resources/gcd-head.zip
+src/test/resources/gcd-v1beta2-rev1-2.1.1.zip
```

Comment:
Can we make a new `.gitignore` with just the ignores that we know we need?

Reply:
Sure. Let me update them.

Reply:
Removed the irrelevant entries.